builder_requests.go

```diff
@@ -135,25 +135,25 @@ func (c *Client) BuildFunction(module *Module, template string, functionBody io.
 	return buildResult, nil
 }
 
-// BuildFunctionString triggers a remote build for the given Module and function string. See also: Client.BuildFunction()
-func (c *Client) BuildFunctionString(module *Module, template, functionString string) (*BuildResult, error) {
-	buf := bytes.NewBufferString(functionString)
-	return c.BuildFunction(module, template, buf)
+// BuildPluginString triggers a remote build for the given Plugin and source code. See also: Client.BuildPlugin()
```

Comment:
```suggestion
// BuildPluginString triggers a remote build for the given plugin and source code. See also: Client.BuildPlugin()
```
Nit based on feedback in other repos.